sql/npc_list.sql

```diff
@@ -3988,7 +3988,6 @@ INSERT INTO `npc_list` VALUES (16982044,'_1es','',0,-11.692,1.690,-144.423,1,40,
 INSERT INTO `npc_list` VALUES (16982045,'_1et','',0,12.066,1.385,144.424,1,40,40,9,0,0,0,6147,0x0200000000000000000000000000000000000000,0,NULL,0);
 INSERT INTO `npc_list` VALUES (16982046,0x14,'',0,-20.726,3.389,-154.231,98311,40,40,18,0,0,2,2051,0x0400000000000000000000000000000000000000,32,NULL,1);
 INSERT INTO `npc_list` VALUES (16982047,'Zhamwaa','Zhamwaa',0,-103.323,0.001,-76.504,8,50,50,0,0,0,0,27,0x01000301AE10AB20AB309F409F50006000700000,32,'TOAU',1);
-INSERT INTO `npc_list` VALUES (16982048,'Moogle','Moogle',64,0.000,0.000,1.500,32769,50,50,0,1,0,0,3,0x0000520000000000000000000000000000000000,0,'TOAU',1);
```

Comment:
Is this the right way to do this? A re-cap in the zone in the future will just place them right back there.

Reply:
Actually seems this is the Moghouse moogle so definitely don't want to remove it.